app/controllers/confirmations_controller.rb

```diff
@@ -11,7 +11,11 @@ def show
       set_flash_message!(:notice, :confirmed)
       if resource.creator?
         sign_in(resource)
-        redirect_to root_path
+        if FeatureFlag.enabled?(:creator_onboarding)
```

Comment:
These if else's are a bit clunky, but will be removed in a few weeks when the feature flag is live.

Reply:
I imagine once the feature flag is removed, we'll still have an if but it will be to check if the logged-in user is the creator?

Reply:
@nickytonline we check if the user is a creator 3 lines up on line 12 of the file, so we could probably remove the if for the feature flag completely :)